Moto/G6/AndroidManifest.xml

```diff
@@ -0,0 +1,10 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+        package="me.phh.treble.overlay.moto.g6"
+        android:versionCode="1"
+        android:versionName="1.0">
+        <overlay android:targetPackage="android"
+                android:requiredSystemPropertyName="ro.vendor.product.name"
+                android:requiredSystemPropertyValue="ali_retail"
```

Comment:
All Moto G6 are called ali_retail, there won't be any variants, like ali_verizon, or ali_india or such stuff?

Reply:
I don't know if there are other version. Would something like `"ali_*"` work or do I have to use some entirely different SystemProperty?

Reply:
I don't know if there are other version. Would something like `"ali_*"` work or do I have to use some entirely different SystemProperty?

Reply:
Also please use fingerprint instead of ro.vendor.product to be pie-vendor proof.
And yes use ali_*
See https://github.com/phhusson/vendor_hardware_overlay/pull/32/commits/97c46901fbd28002d3d028f9575e285bfb1a190c as an example

Reply:
Done. (5ec4807)
(Though I haven't tested it yet. I'll note it here once I get around to that.)